### DISCUSSION TITLE
Telemetry problem

### DISCUSSION THREAD

**Discussion by pat1300** [iNavFlight/inav#10173] [Q&A] [answered]
![IMG_20240622_220734](https://github.com/iNavFlight/inav/assets/11511063/43d12f28-500a-436c-8733-19bf8f850ae1)
Hi ,
I have an issue with some telemetry sensors reported by iNav.
My configuration:
FC SpeedyBee Wing  F405 running INAV 7.1.2
Receiver: Flysky FS IA-6B
Tx: RadioMaster TX16S running EdgeTX 2.9.4 Providence (Internal Multi module V.1.3.3.7)

In the TX, without telemetry connected between FC and RX, I correctly get values for 6 sensors ( RSSI, …)
When I connect FC TX2 (Telemetry UART2, protocol IBUS) to sense pin of receiver Flysky FS IA-6B, I get in the TX 16 more telemetry sensors ( total : 21 ). 
Some are correct ( like the voltage, current  of the battery connected to the FC), but most have zero values, for example the GPS coordinates, speed, …
However, I am sure that the GPS is working fine, because when I connect the iNAV configurator to the FC ( using a PC, or using the SpeedyBee app that connects using BLE to the FC), I see correct longitude and latitude values in the FC.
I tried to set different values of “ibus_telemetry_type” in inav, without success. ( The file telemetry.md in docs describes the sensor id that should be transmitted by iNav, but I get different values when discovering sensors in the TX) 

Any idea why those sensors are not reported correctly to the TX? Any recommendation for ibus_telemetry_type ? ( or do I modify another setting?)

Thanks for your help,
Patrick - Belgium

**Comment by stronnag** [comment, marked as answer]
Your problem is with EdgeTX / OpenTX; they do not understand the extended fields provided by INAV.

INAV IBUS telemetry will be displayed by a tool that understands it, either a Flysky TX with custom firmware as described in the INAV  documentation or a ground station that understands the INAV flavour of IBUS telemetry.

**Reply by pat1300** [reply]
Thanks!
